docs/job.md

```diff
@@ -126,18 +128,17 @@ The overall job state can be added to the job list response:
       "status":"running"
     },
     …
-  ],
-  "next":"/api/v1/job?limit=100&start=3&fields=status",
-  "previous":"/api/v1/job?limit=100&start=1&fields=status"
+  ]
 }
 ```
 
 ### GET query parameters for listing jobs
 
 - `compendium_id` - Comma-separated list of related compendium ids to filter by.
 - `start` - List from specific search result onwards. 1-indexed. Defaults to 1.
```

Comment:
Again, what does "1-indexed" mean? Please remove and re-check how MongoDB phrases this.